cmd/skuba/cluster/upgrade.go

```diff
@@ -60,3 +62,17 @@ func newUpgradePlanCmd() *cobra.Command {
 		Args: cobra.NoArgs,
 	}
 }
+
+func newUpgradeConfigCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "config",
+		Short: "Upgrades the local configuration",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := ssh.CriMigrate(); err != nil {
```

Comment:
this is very misleading. You are saying migrating local configuration, yet this function run things on the node.

For me the upgrade localconfig should be dealing with just local things, while the things happening on target nodes can be dealt with during `node upgrade`.

Reply:
This function does not touch files on remote nodes. It just updates the local configuration. Thus, I see two things:

1. It should not go into the `ssh` package, as you said in another comment.
2. We may want to change the subcommand name from `config` to `localconfig`, so to reinforce the idea of changes only happening locally, as it has been suggested. What do you think?